roles/jd-server/README.md

```diff
@@ -0,0 +1,27 @@
+# JD Server
+
+The JD Server is a Sv2 proxy, It is also the entity responsible for Pool-side block propagation in case of valid blocks found by miners connected to the pool (who are using the Job Declaration Protocol).
+
+## Setup
+
+### Configuration File
+
+The configuration file contains the following information:
+
+1. The SRI Pool information which includes the SRI Pool authority public key (`authority_public_key`), the SRI Pool authority secret key (`authority_secret_key`), along with its certificate validity (`cert_validity_sec`). In addition to this, it contains the address which it will use to listen to new connection from downstream roles (`listen_jd_address`) and the list of uncompressed pubkeys for coinbase payout (`coinbase_outputs`).
+2. The RPC config for mempool config where are (`core_rpc_url`, `core_rpc_port`, `core_rpc_user`, `core_rpc_pass`), and the time interval for the JDS mempool update in form of (`unit`,`value`)
+
+### Run
+
+There are two files when you cd into roles/jd-server/config-examples/
+
+1. jds-config-hosted-example.toml is community hosted server
```

Comment:
In order to communicate with bitcoin node via rpc, it is strongly seggested (if not mandatory) to run the node locally with respect to the JD server. So, imo it should be specified that in each case bitcoin node is "local".

Reply:
@lorbax in this case we shouldn't even be hosting a TP for the community?

cc @GitGab19

Reply:
or is it possible to have yet another Bitcoin Core node that is not TP, but is used for mempool RPC?